src/sql/src/query_model/hir.rs

```diff
@@ -130,4 +207,24 @@ impl FromHir {
         }
         panic!("column not found")
```

Comment:
Improve error message here - explain why the column is not found and provide the `BoxId` as context.

Reply:
This should be `unreacheable` instead. There is no reason for a column not to be found other than a bug.